src/Neo.CLI/CLI/MainService.Contracts.cs

```diff
@@ -133,21 +133,28 @@ private void OnInvokeCommand(UInt160 scriptHash, string operation, JArray? contr
         {
             var gas = new BigDecimal(maxGas, NativeContract.GAS.Decimals);
             Signer[] signers = Array.Empty<Signer>();
-            if (!NoWallet() && sender != null)
+            if (!NoWallet())
             {
-                if (signerAccounts == null)
-                    signerAccounts = new UInt160[1] { sender };
-                else if (signerAccounts.Contains(sender) && signerAccounts[0] != sender)
-                {
-                    var signersList = signerAccounts.ToList();
-                    signersList.Remove(sender);
-                    signerAccounts = signersList.Prepend(sender).ToArray();
-                }
-                else if (!signerAccounts.Contains(sender))
+                var defaultAccount = CurrentWallet!.GetDefaultAccount();
+                if (sender == null && defaultAccount != null)
```

Comment:
If sender != null we don't need the variable, could you put it inside the condition?